java/google/registry/rde/imports/RdeDomainImportAction.java

```diff
@@ -121,6 +123,13 @@ private RdeImportUtils getImportUtils() {
       return importUtils;
     }
 
+    private DnsQueue getDnsQueue() {
```

Comment:
The reason this is here is so that the DnsQueue can be retrieved after the mapper is unmarshalled. When the deserialization happens, the constructor is not invoked.